lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java

```diff
@@ -82,34 +77,35 @@
      * @param width
      *            the width of the vertex
      * @param height
-     *            the height of the vertex
-     * @param scale
+     *            the amount of sequences going through this node
+     * @param horizontalScale
      *            the resize factor of the vertex
+     * @param verticalScale
+     *            the vertical scale factor for this vertex
      * @param color
      *            the color of the vertex
      */
     public VertexView(final String string, final Point2D topLeftPoint,
-            final double width, final double height, final double scale,
+            final double width, final double height,
```

Comment:
Too many arguments now, you could combine `width` and `height`, same for `horizontalScale` and `verticalScale`